Gemfile.lock

```diff
@@ -74,10 +74,10 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: f550220a7c2cbe327f041156aa8befd0f6b5b77e
+  revision: a6815eb11dde49c728182e9fd2370410a37ad005
   branch: master
   specs:
-    vets_json_schema (24.13.0)
+    vets_json_schema (24.13.3)
```

Comment:
Matches https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/package.json#L3 👍